### ISSUE TITLE
Try out Rectified Adam, a new optimizer less sensitive to learning rates

### ISSUE THREAD

**Issue by tabacof** [Accenture/AmpliGraph#130]
**Background and Context**
https://medium.com/@lessw/new-state-of-the-art-ai-optimizer-rectified-adam-radam-5d854730807b

TL;DR: Performance similar to ADAM,  but a lot less sensitive to the learning rate choice, something we struggle with when training KGE.

**Description**
Keras unnofficial implementation: 
https://github.com/CyberZHG/keras-radam
https://github.com/niley1nov/keras_rnadam

I could not find any TF 1.13 / 1.14 implementation.

**Comment by tabacof** [comment]
TF implementation: https://github.com/taki0112/RAdam-Tensorflow

**Comment by tabacof** [comment]
Using the implementation above and the same parameters as used in the predictive_performance.py script, I got the following results:

![image](https://user-images.githubusercontent.com/2151702/64241411-9e551500-cefb-11e9-9da2-da92cff00eab.png)

The performance of RAdam was slightly better than Adam in WN18RR, but it was virtually the same in FB15K-237. The training times, however, were clearly worse. 

Note that training times are highly dependent on early stopping, which was configured in the same manner used in the predictive_performance.py script.

Considering that there would be one extra dependency, RAdam does not seem worth the trouble and I am closing this issue for now. If RAdam is eventually incorporated into the official TensorFlow, we should consider adding it as one other optimizer.
